MediaGalleryUi/Model/SearchCriteria/CollectionProcessor/FilterProcessor/EntityType.php

```diff
@@ -41,6 +42,13 @@ public function apply(Filter $filter, AbstractDb $collection): bool
     {
         $value = $filter->getValue();
         if (is_array($value)) {
+            if (in_array(self::NOT_USED, $value)) {
```

Comment:
We may have to change this approach, this is conflicting with other options on the same filter, instead of using `nin` I think we should `in` condition and get the asset ids that don't have a relation with content.

The behavior here should be that if you select `Products` and `Not used anywhere` the user should see both, assets used in products or assets not used anywhere.

Reply:
ok thanks, agree :+1: